runhouse/resources/hardware/cluster.py

```diff
@@ -741,17 +741,37 @@ def connect_server_client(self, force_reconnect=False):
                 system=self,
             )
 
-    def status(self, resource_address: str = None):
-        """Loads the status of the Runhouse daemon running on the cluster."""
+    def status(self, resource_address: str = None, send_to_den: bool = False):
+        """Load the status of the Runhouse daemon running on a cluster."""
+
         # Note: If running outside a local cluster need to include a resource address to construct the cluster subtoken
         # Allow for specifying a resource address explicitly in case the resource has no rns address yet
         if self.on_this_cluster():
-            status = obj_store.status()
+            status_res = obj_store.status(send_to_den=send_to_den)
         else:
-            status = self.call_client_method(
+            status_res = self.call_client_method(
                 "status",
                 resource_address=resource_address or self.rns_address,
+                send_to_den=send_to_den,
             )
+
+        # parsing the status response, depending on the send_to_den flag. If send_to_den, both the status and the
+        # response to den will be retrieved.
+        status, den_resp = status_res if len(status_res) == 2 else (status_res, None)
```

Comment:
would prefer if the status calls return types stayed consistent, whether its thru the object store or HTTP call. we can just have den_resp be None in that case - meaning something like: 
`status_res, _ = obj_store.status(send_to_den=send_to_den)`